suite-native/navigation/src/routes.ts

```diff
@@ -143,7 +143,8 @@ export enum SettingsStackRoutes {
     SettingsFAQ = 'SettingsFAQ',
     SettingsCoinEnabling = 'SettingsCoinEnabling',
     SettingsDeviceChecks = 'SettingsDeviceChecks',
-    TurnOffFirmwareAuthenticityCheckModal = 'TurnOffFirmwareAuthenticityCheckModal',
+    TurnOffDeviceAuthenticityCheck = 'TurnOffDeviceAuthenticityCheck',
+    TurnOffFirmwareAuthenticityCheck = 'TurnOffFirmwareAuthenticityCheck',
```

Comment:
Since you can also turn the checks on, I'm not sure that having `TurnOff` in the name is the best choice. Maybe `Toggle`, `Switch`, `Configure`, ...? I'd then propagate to all relevant classes.

Not a big deal, feel free to ignore..

Reply:
Where do we do that? Noticed this?
<img width="356" alt="image" src="https://github.com/user-attachments/assets/3afd6f36-8d1b-46bb-9f4c-b003fc55a4b4" />

Reply:
In conversations and github issues...